modules/scanning-images-in-a-remote-cluster.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * cli/image-scanning-by-using-the-roxctl-cli.adoc
+
+:_content-type: PROCEDURE
+[id="scanning-images-in-a-remote-cluster_{context}"]
+= Scanning images in a remote cluster
+
+.Prerequisites
+
+* You have created an image using a Docker file and added it to the {ocp} integrated registry.
```

Comment:
Would recommend changing these instructions to focus on general usage of `roxctl image scan` (as opposed to being specific to the OCP integrated registry).  And add a note or similar that the `cluster` flag can be used to direct the scan to specific clusters in support of scanning the OCP integrated registry.